packages/ui/bin/buildStyleVars/runner.js

```diff
@@ -1,3 +1,4 @@
+const fs = require('fs')
```

Comment:
I don't think it's used
```suggestion
const fs = require('fs')
```